dashboard/app/models/regional_partner.rb

```diff
@@ -15,6 +15,14 @@
 
 class RegionalPartner < ActiveRecord::Base
   belongs_to :contact, class_name: 'User'
+
+  has_many :regional_partner_organizers
+  has_many :organizers,
+    class_name: 'User',
+    through: :regional_partner_organizers
```

Comment:
Oh, perfect! I tried to find out what we called these folk yesterday, but never got an answer. Program managers sounds perfect, thanks!